lib/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# evmone: Fast Ethereum Virtual Machine implementation
+# Copyright 2024 The evmone Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(crypto STATIC)
+add_library(evmone::crypto ALIAS crypto)
+target_compile_features(crypto PRIVATE cxx_std_20)
+target_link_libraries(crypto PUBLIC evmc::evmc_cpp)
+target_sources(
+    crypto PRIVATE
```

Comment:
I think creating the `evmone::crypto` library may be needed, but I was rather thinking of renaming `evmoe::precompiles` to `evmone::crypto`. Do we need this now?

Reply:
I meant renaming `evmone::precompiles` library `evmone::crypto`. For you just add `evmone::precompiles` as the dependency. It depends on it implicitly already.